### PR TITLE
#5: Project Danbooru.

### PR DESCRIPTION
**Pull request by tangalbert919**
To anyone who asks, this is a real pull request. Yes, this is happening.

I will get around to the safe stuff later.

### CODE REVIEW

File: cogs/music.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def __init__(self, bot):
         self.bot = bot
 
         if not hasattr(bot, 'lavalink'):
-            lavalink.Client(bot=self.bot, password=self.bot.lavapass, loop=self.bot.loop, ws_port=1337, shard_count=len(bot.shards))
+            lavalink.Client(bot=self.bot, password=bot.settings.lavapass, loop=self.bot.loop, ws_port=1337, shard_count=len(bot.shards))
```
Comment (zspri) [inline cogs/music.py]: Still needs to have 'self'

**Review by zspri — COMMENTED** [review]
Idk what this is really but there's a single error as far as I can see. (edit: guess it's not an error)

File: cogs/music.py (hunk 1 above)
Comment (kanielyaniel) [reply cogs/music.py]: @devakira False, `bot` is initialized in the snippet of code but using `self` would be a better practice.

Comment (zspri) [reply cogs/music.py]: Oh well.

**Review by kanielyaniel — APPROVED** [review]

**Pull request merged**